src/mappings/components/ExecutorMapping.tsx

```diff
@@ -117,7 +117,7 @@ export const ExecutorMapping: InspectableMapping<
   },
   docsInfo: {
     description:
-      'An %s technology or environment which Jobs execute their steps inside of.',
+      'An %s is a technology or environment in which Jobs execute their steps inside of.',
```

Comment:
Or maybe this should be "An Executor is a technology or environment which Jobs executor their steps inside of"
(no IN in middle)

Definitely I feel like it was missing "is a" for clarity in the beginning 😁 welcome counter opinions!